packages/webapp-libs/webapp-finances/src/routes/subscriptions/subscriptions.component.tsx

```diff
@@ -1,43 +1,65 @@
+import { PageHeadline } from '@sb/webapp-core/components/pageHeadline';
+import { PageLayout } from '@sb/webapp-core/components/pageLayout';
+import { Separator } from '@sb/webapp-core/components/separator';
 import { FormattedMessage } from 'react-intl';
 
 import { useActiveSubscriptionDetails } from '../../components/activeSubscriptionContext';
 import { PaymentMethodContent } from './paymentMethod.content';
 import { SubscriptionsContent } from './subscriptions.content';
-import { Container, Header, Section, Subheader } from './subscriptions.styles';
```

Comment:
I think we can remove this style file

Reply:
This file contain styles that are used by SubscriptionsContent component which is not done yet